src/common/store/pages/audio-rewards/slice.ts

```diff
@@ -56,6 +56,7 @@ type RewardsUIState = {
   userChallengesOverrides: Partial<
     Record<ChallengeRewardID, Partial<UserChallenge>>
   >
+  disbursedChallenges: Partial<Record<ChallengeRewardID, string[]>>
```

Comment:
nitty: might be nice to leave a comment / or type `Specifier = string` to know what type the array value is

Reply:
totally agree. actually decided to type specifier everywhere we use it